test/helpers/client.js

```diff
@@ -41,6 +44,24 @@ class Client {
         this._respondWithCallback(JSON.parse(data));
     }
 
+    _checkError(error) {
+        // Assumes that an instance of the server is not running, so lets start one.
+        if (error.errno === "ECONNREFUSED") {
+            if (!this.child) {
+                this.child = fork('main.js', ['grunt', '--debug', 'silly'])
```

Comment:
Why the need to fork here? Also this will make Concierge think you have required it rather than called it (so some modules might fail)